man/man8/zfs-load-key.8

```diff
@@ -166,6 +166,29 @@ will become one. Alternatively, the
 .Fl i
 flag may be provided to cause an encryption root to inherit the parent's key
 instead.
+.Pp
+Changing the user's key does not change the underlying master key used to
+encrypt the dataset.  Accordingly, if the master key has been compromised,
+both existing data and new data written after the
+.Nm zfs Cm change-key
+would be readable, so this command must not be relied upon to address a
+master key compromise.
+.Pp
```

Comment:
How about adding something here to explain "master key compromise".  E.g.: `The master key can be compromised if the attacker obtains a previous user key and the corresponding wrapped master key.  The wrapped master key is stored ondisk, and may be available via forensic analysis even after the change-key operation.`

Reply:
I've rewritten this, merging in your notes. See what you think now.

Reply:
Thanks, this looks good to me!